doc/classes/EditorInterface.xml

```diff
@@ -285,6 +285,23 @@
 				See also [method Window.set_unparent_when_invisible].
 			</description>
 		</method>
+		<method name="popup_dialog_property_selector">
+			<return type="void" />
+			<param index="0" name="object" type="Object" />
+			<param index="1" name="callback" type="Callable" />
+			<param index="2" name="type_filter" type="PackedInt32Array" default="PackedInt32Array()" />
+			<description>
+				Pops up an editor dialog for selecting properties from [param object]. The [param callback] must take an [Array] of [NodePath]s. It is [method Callable.call_deferred] on an array containing the selection (see [method NodePath.get_as_property_path]) if an item is selected and [method Callable.call_deferred] on an empty array if the dialog is cancelled.
+			</description>
```

Comment:
Also I think you could add some example code for both methods.

Reply:
I'll add an example in GDScript, but I don't use C# so I might need some help translating it to have a valid C# example as well.